src/main/java/study/IntegerUtil.java

```diff
@@ -0,0 +1,17 @@
+package study;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class IntegerUtil {
+
+    private IntegerUtil() {
+    }
```

Comment:
빈 생성자는 아래와 같이 한 줄로 붙여 써도 무방합니다. 
```suggestion
    private IntegerUtil() {}